### PR TITLE
Task-50506: Display warning message to remember when the space has a …

### PR DESCRIPTION
**Pull request by lbengayed**
…redactor

### CODE REVIEW

File: notes-webapp/src/main/webapp/vue-app/notes/components/NotesOverview.vue [hunk 1]
```diff
@@ -320,6 +322,9 @@ export default {
       const uris = eXo.env.portal.selectedNodeUri.split('/');
       return uris[uris.length - 1];
     },
+    alertWarningDisplayed(){
```
Comment (mdiha) [inline notes-webapp/src/main/webapp/vue-app/notes/components/NotesOverview.vue:325]: Can't you just check it with data property 'alert' ?

File: notes-webapp/src/main/webapp/vue-app/notes/components/NotesOverview.vue [hunk 2]
```diff
@@ -166,6 +166,8 @@
     <v-alert
       v-model="alert"
       :type="type"
+      :icon="type === 'warning' ? 'mdi-alert-circle' : ''"
+      @input="onclose"
```
Comment (mdiha) [inline notes-webapp/src/main/webapp/vue-app/notes/components/NotesOverview.vue:170]: @input Not needed IMO

File: notes-webapp/src/main/webapp/vue-app/notes/components/NotesOverview.vue [hunk 3]
```diff
@@ -440,8 +445,12 @@ export default {
         console.error('Error when getting note', e);
         this.existingNote = false;
       }).finally(() => {
-        if (!this.note.canManage){
-          this.$root.$emit('show-alert', {type: 'warning',message: this.$t('notes.alert.warning.label.notification')});
+        if (!this.note.canManage && !this.alertWarningDisplayed){
```
Comment (mdiha) [inline notes-webapp/src/main/webapp/vue-app/notes/components/NotesOverview.vue:449]: The local storage needs to be set here.

File: notes-webapp/src/main/webapp/vue-app/notes/components/NotesOverview.vue [hunk 4]
```diff
@@ -471,6 +480,13 @@ export default {
         console.error('Error when getting note', e);
         this.existingNote = false;
       }).finally(() => {
+        if (!this.note.canManage && !this.alertWarningDisplayed){
```
Comment (mdiha) [inline notes-webapp/src/main/webapp/vue-app/notes/components/NotesOverview.vue:484]: The local storage needs to be set here too.

File: notes-webapp/src/main/webapp/vue-app/notes/components/NotesOverview.vue [hunk 5]
```diff
@@ -559,11 +575,16 @@ export default {
...
       this.message=message.message;
       this.type=message.type;
       this.alert = true;
-      window.setTimeout(() => this.alert = false, 5000);
+      if (!keepAlert) {
+        window.setTimeout(() => this.alert = false, 5000);
+      }
+    },
+    onclose() {
+      localStorage.setItem(`displayAlert-${this.spaceDisplayName}`, 'already_display');
```
Comment (mdiha) [inline notes-webapp/src/main/webapp/vue-app/notes/components/NotesOverview.vue]: What if the space's display name changes ? Better rename it according to an unchangeable attribute

File: notes-webapp/src/main/webapp/vue-app/notes/components/NotesOverview.vue (hunk 3 above)
Comment (lbengayed) [reply notes-webapp/src/main/webapp/vue-app/notes/components/NotesOverview.vue:449]: we need to set the local storage when clicking on the close icon, here we need only test if the alert warning is already displayed or not

Comment (mdiha) [reply notes-webapp/src/main/webapp/vue-app/notes/components/NotesOverview.vue:449]: Okay

File: notes-webapp/src/main/webapp/vue-app/notes/components/NotesOverview.vue (hunk 5 above)
Comment (lbengayed) [reply notes-webapp/src/main/webapp/vue-app/notes/components/NotesOverview.vue]: done!

**Review by SaraBoutej — APPROVED** [review]

**Pull request merged**